lib/suse/connect/system_serializer.rb

```diff
@@ -47,6 +48,17 @@ def products
     end
   end
 
+  def online_at
+    object.system_uptimes.map do |systemuptime|
+      payload = {
+        system_id: systemuptime.system_id,
```

Comment:
Afaics that parameter is not used in https://github.com/SUSE/happy-customer/pull/7339/, what's the reason for including it into the payload?

Reply:
fixed